lib/cassandra.js

```diff
@@ -625,10 +611,29 @@ Cassandra.prototype.fromColumnValue = function(prop, val) {
         break;
       case 'List':
       case 'Array':
+        if (typeof val === 'string') {
+          try {
+            val = JSON.parse(val);
+          } catch(e) {}
+        }
+        if (Array.isArray(val)) {
+          var ret = [];
+          val.forEach(function(v) {
+            try {
+              ret = ret.concat(JSON.parse(v));
```

Comment:
If the val is already an array, why do we try to parse items inside the array again? I don't think we need to handle such mixed usage.